generated/python/gapic-google-cloud-pubsub-v1/google/cloud/gapic/pubsub/v1/publisher_client.py

```diff
@@ -213,8 +204,7 @@ def __init__(self,
             client_config,
             config.STATUS_CODE_NAMES,
             metrics_headers=metrics_headers,
-            bundle_descriptors=self._BUNDLE_DESCRIPTORS,
```

Comment:
_Technically_ this is a breaking change, so by convention we'd bump to `0.16.0`. In practice, I'm fairly sure no one is using GAPIC bundling, which is a complicated and unpublicized feature. What are your thoughts here?